src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -568,7 +568,27 @@ export class VolumesListTableConfig implements InputTableConf {
     
                   }, (e_res) => {
                     this.loader.close();
-                    this.dialogService.errorReport(T("Error deleting dataset ") + "<i>" + row1.path + "</i>.", e_res.reason, e_res.stack);
+                    if (e_res.reason.indexOf('Device busy') > -1) {
+                      this.dialogService.confirm(T('Device Busy'), T('Do you want to force delete dataset ') + "<i>" + row1.path + "</i>?", false, T('Force Delete')).subscribe(
```

Comment:
```suggestion
                      this.dialogService.confirm(T('Device Busy'), T('Force deletion of dataset ') + "<i>" + row1.path + "</i>?" + T('System processes using this dataset might be affected.'), false, T('Force Delete')).subscribe(
```

Reply:
Can a caution as to why the user might not want to force the delete be worked in?